schema/core/v2.0.0/metadata.yaml

```diff
@@ -594,6 +612,9 @@ classes:
       is_visualization_default:
         exact_mappings:
           - cdp-common:annotation_source_file_is_visualization_default
+      is_portal_standard:
+        exact_mappings:
+          - cdp-common:annotation_source_file_is_portal_standard
```

Comment:
We had a discussion about the `is_portal_standard` field the other day and iirc It's only supposed to be present on `Tomogram` and `Alignment`

Reply:
cc @uermel

Reply:
remove from v1 and v2